geocoding/location-service/src/app/geocoder_suggester.py

```diff
@@ -44,3 +48,18 @@ def suggest(self, request):
                 return filtered_suggestions
         logger.debug(f"No suggestions for query {request['q']}")
         return []
+
+    def suggest_admin1(self, admin0):
+        if admin0 not in adm1ParsedData:
+            return None
+        return adm1ParsedData[admin0]
```

Comment:
Could be `return adm1ParsedData.get(admin0)` (default return from `get` is `None`)